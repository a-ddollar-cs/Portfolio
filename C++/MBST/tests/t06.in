42	92	
17	32	1
0	1	2
8	11	3
40	41	4
14	41	5
9	41	6
20	26	7
21	33	8
13	21	9
13	14	10
5	21	11
14	18	12
16	23	13
7	16	14
3	19	15
3	4	16
17	27	17
4	30	18
21	31	19
3	5	20
27	31	21
15	21	22
1	38	23
13	41	24
3	24	25
13	18	26
1	2	27
7	9	28
0	6	29
8	19	30
13	30	31
12	24	32
7	28	33
26	39	34
2	3	35
9	25	36
34	41	92
14	25	37
33	39	38
5	7	39
22	24	40
17	20	41
10	26	42
34	39	43
36	40	44
37	41	45
29	38	46
32	41	47
24	41	48
28	35	49
12	22	50
15	20	51
21	39	52
14	15	53
15	17	54
20	22	55
27	34	56
21	37	57
27	29	58
19	36	59
34	38	60
15	18	61
29	34	62
31	35	63
33	41	64
39	40	65
14	37	66
11	12	67
28	38	68
27	36	69
2	23	70
2	10	71
7	8	72
17	28	73
32	38	74
38	40	75
28	41	76
32	33	77
33	36	78
31	36	79
37	38	80
11	16	81
12	41	82
12	20	83
12	13	84
26	30	85
26	36	86
32	34	87
35	37	88
37	39	89
33	34	90
35	40	91
