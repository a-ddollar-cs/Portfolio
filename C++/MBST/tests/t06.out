28 35 49
